standards/SPIFFE_Trust_Domain_and_Bundle.md

```diff
@@ -0,0 +1,290 @@
+5.2. [Authenticating the Bundle Endpoint](#52-authenticating-the-bundle-endpoint)  
+5.2.1. [Web PKI](#521-web-pki)  
+5.2.2. [SPIFFE Authentication](#522-spiffe-authentication)  
+6\. [Security Considerations](#6-security-considerations)  
+6.1. [Securing SPIFFE Bundle Transmission](#61-securing-spiffe-bundle-transmission)  
+6.2. [SPIFFE Bundle Refresh Hint](#62-spiffe-bundle-refresh-hint)  
+6.3. [Using Web PKI on a SPIFFE Bundle Endpoint](#63-using-web-pki-on-a-spiffe-bundle-endpoint)  
+6.4. [Reusing Cryptographic Keys Across Trust Domains](#64-reusing-cryptographic-keys-across-trust-domains)  
+Appendix A. [SPIFFE Bundle Example](#appendix-a-spiffe-bundle-example)  
+Appendix B. [Bundle Endpoint Authentication Examples](#appendix-b-bundle-endpoint-authentication-examples)  
+Appendix B.1. [Web PKI-based Endpoint Authentication](#appendix-b1-web-pki-based-endpoint-authentication)
+Appendix B.2. [SPIFFE-based Endpoint Authentication](#appendix-b2-spiffe-based-endpoint-authentication)
+
+## 1. Introduction
+SPIFFE trust domains represent the basis by which a SPIFFE ID is qualified, indicating the realm or authority under which any given SPIFFE ID has been issued. They are backed by an issuing authority, which is tasked with managing the issuance of SPIFFE identities within its respective trust domain. While the name of a trust domain consists of a simple human-readable string, it is also necessary to express the cryptographic keys in use by the trust domain's issuing authority, enabling others to validate the identities it issues. These keys are expressed as a "SPIFFE bundle", which goes hand in hand with the trust domain that it represents.
+
+This specification defines the nature and semantics of both SPIFFE trust domains and bundles. It also defines the "bundle endpoint", a mechanism by which SPIFFE bundles may be communicated between trust domains.
+
+## 2. Trust Domains
+A SPIFFE trust domain is an identity namespace which is backed by an issuing authority with a set of cryptographic keys. Together, these keys serve as the cryptographic anchor for all identities residing in the trust domain.
+
+Trust domains have a 1:N relationship with the keys that back them. A single trust domain may be represented by multiple keys and key types. For example, the former may be leveraged during root rotation, while the latter is necessary in avoiding multi-protocol attacks should more than one SVID type be in use.
+
+It should be noted that while it is possible to share cryptographic keys amongst many trust domains, it is strongly encouraged to maintain a one-to-one mapping between a trust domain and any given authoritative key. Key reuse can degrade trust domain isolation (e.g. between staging and production) and introduces additional security challenges (e.g. requiring a name constraint system for secondary issuers). Please see the [Security Considerations](#6-security-considerations) section for more information on this topic.
```

Comment:
```suggestion
It should be noted that while it is possible to share cryptographic keys amongst many trust domains, we strongly advise that each authoritative key be used in a single trust domain. Key reuse can degrade trust domain isolation (e.g. between staging and production) and introduces additional security challenges (e.g. requiring a name constraint system for secondary issuers). Please see the [Security Considerations](#6-security-considerations) section for more information on this topic.
```

- passive voice sounded strange
- it's not 1:1; we just said in the previous paragraph that it's 1:N